matrix/common/etl/transformers/cell_expression.py

```diff
@@ -16,6 +16,8 @@
 from matrix.common.etl.dcp_zarr_store import DCPZarrStore
 from matrix.common.exceptions import MatrixException
 
+EMPTYDROPS_MIN_COUNT = 100
```

Comment:
Can we make this a class variable in the class?

Reply:
Yeah, will fix.